cmd/modern/root.go

```diff
@@ -113,4 +114,12 @@ func (c *Root) addGlobalFlags() {
 		Shorthand:  "v",
 		Usage:      "Log level, error=0, warn=1, info=2, debug=3, trace=4",
 	})
+
+	c.AddFlag(cmdparser.FlagOptions{
+		String:        &c.inputType,
+		DefaultString: "yaml",
+		Name:          "InputFile",
+		Shorthand:     "i",
+		Usage:         "output type (yaml, json or xml)",
```

Comment:
this implies 3 different concepts - 
1. input type
2. input file name
3. Output type

Reply:
I think this is a copy and paste error (from outputType above)